package.json

```diff
@@ -1,37 +1,45 @@
 {
-  "name": "jellyfin-theater",
-  "productName": "Jellyfin Theater",
+  "name": "jellyfin-desktop",
   "version": "3.0.5",
-  "description": "Jellyfin Theater Made With Electron",
+  "description": "Jellyfin Desktop made with Electron",
   "main": "main.js",
   "scripts": {
     "start": "electron main.js",
-    "package-mac": "electron-packager . --overwrite --platform=darwin --arch=x64 --icon=icon.ico --prune=true --out=release",
-    "package-win": "electron-packager . Jellyfin-Theater --overwrite --asar=true --platform=win32 --arch=ia32 --icon=icon.ico --prune=true --out=release --version-string.CompanyName=CE --version-string.FileDescription=CE --version-string.ProductName=\"Jellyfin Theater\"",
-    "package-linux": "electron-packager . Jellyfin-Theater --overwrite --asar=true --platform=linux --arch=x64 --icon=icon.ico --prune=true --out=release"
+    "package-mac": "electron-builder -m",
+    "package-win": "electron-builder -w",
+    "package-linux": "electron-builder -l"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/jellyfin/jellyfin-theater-electron.git"
+    "url": "git+https://github.com/jellyfin/jellyfin-desktop.git"
   },
   "keywords": [
     "Electron",
     "Jellyfin",
     "Theater",
     "Media Client"
   ],
-  "author": "GitHub",
+  "author": {
+    "name": "Jellyfin"
+  },
   "license": "GPL-2.0-only",
   "bugs": {
-    "url": "https://github.com/jellyfin/jellyfin-theater-electron/issues"
+    "url": "https://github.com/jellyfin/jellyfin-desktop/issues"
   },
-  "homepage": "https://github.com/jellyfin/jellyfin-theater-electron#readme",
+  "homepage": "https://github.com/jellyfin/jellyfin-desktop#readme",
   "devDependencies": {
-    "electron": "^3.0.0"
+    "@typescript-eslint/eslint-plugin": "^2.29.0",
+    "@typescript-eslint/parser": "^2.29.0",
+    "electron": "^3.0.0",
```

Comment:
We can probably upgrade this to Electron 8, right?

Reply:
Be my guest

Reply:
Handled in the 3rd PR